# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../commons-codec/licensedir.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- commons-codec-1.3/build.xml	2004-05-24 01:30:33.000000000 +0200
+++ commons-codec-1.3-patch/build.xml	2005-07-26 21:56:39.104650128 +0200
@@ -90,14 +90,14 @@
     </target>
     <target name="dist" depends="compile,javadoc" description="Create binary distribution">
         <mkdir dir="${dist.home}"/>
-        <copy file="../LICENSE" todir="${dist.home}"/>
+        <copy file="LICENSE.txt" todir="${dist.home}"/>
         <copy file="${basedir}/RELEASE-NOTES.txt" todir="${dist.home}"/>
         <antcall target="jar"/>
     </target>
     <target name="jar" depends="compile" description="Create jar">
         <mkdir dir="${dist.home}"/>
         <mkdir dir="${build.home}/classes/META-INF"/>
-        <copy file="../LICENSE" tofile="${build.home}/classes/META-INF/LICENSE.txt"/>
+        <copy file="LICENSE.txt" tofile="${build.home}/classes/META-INF/LICENSE.txt"/>
         <jar jarfile="${dist.home}/${final.name}.jar" basedir="${build.home}/classes" manifest="${build.home}/conf/MANIFEST.MF"/>
     </target>
     <target name="install-jar" depends="jar" description="--> Installs jar file in ${lib.repo}">
